modules/get-started/pages/whats-new.adoc

```diff
@@ -6,6 +6,29 @@ This topic includes new content added from version 4.29.0 onwards.
 
 For a full list of product updates, and details of updates prior to version 4.29.0, see the https://github.com/redpanda-data/connect/blob/main/CHANGELOG.md[Redpanda Connect changelog^]. For new Cloud-specific features, see xref:redpanda-cloud:get-started:whats-new-cloud.adoc[].
 
+== Version 4.60.0
+
+=== Component updates
+
+The 4.60.0 release adds support for the following fields:
+
+* NATS components
+** `tls_handshake_first` - Enhanced TLS configuration option for NATS connections that require the initial TLS handshake to be performed before sending the NATS INFO protocol message. This can be required when connecting to some NATS servers that expect TLS to be established immediately after connection, before any protocol negotiation.
```

Comment:
dang. "This can be required..." I know this touches on almost every update/context here, but it I think it can/should be more clear/definitive.